src/helpers/dimensions.ts

```diff
@@ -255,6 +255,20 @@ export function getHeadlinesDimensions(): AFM.IDimension[] {
     return [{ itemIdentifiers: ["measureGroup"] }];
 }
 
+export function getGeoChartDimensions(
+    mdObject: VisualizationObject.IVisualizationObjectContent,
+): AFM.IDimension[] {
+    const { attributes = [] }: AFM.IAfm = convertBucketsToAFM(mdObject.buckets);
+    return [
+        {
+            itemIdentifiers: [MEASUREGROUP],
+        },
+        {
+            itemIdentifiers: attributes.map((attr: AFM.IAttribute) => attr.localIdentifier),
```

Comment:
how to distinguish location and segment by dimension?
- location attribute should be same dimension with `MEASUREGROUP`
- segment by attribute should be on other dimension

Reply:
updated